src/core/src/bootstrap/Constants.py

```diff
@@ -102,6 +102,28 @@ class ConfigSettings(EnumBackport):
     # patch versions for healthstore when there is no maintenance run id
     PATCH_VERSION_UNKNOWN = "UNKNOWN"
 
+    # Strings used in perf logs
+    class PerfLogTrackerParams:
+        TASK = "task"
+        TASK_STATUS = "task_status"
+        PACKAGE_MANAGER = "package_manager"
+        NUMBER_OF_TRIALS = "number_of_trials"
```

Comment:
Non-blocker to this PR, can be addressed trivially in a second PR: retry_count is a standard phrase over number_of_trials

Reply:
Sure, I will take care of this in a separate PR. Completing this PR.